### PR TITLE
V1

### PR DESCRIPTION
**Pull request by demitchell14**
`So first things first, this is a ridiculous pull, and I know it. I am sorry, but I love you.`

# V1 release of the Modmail Web App

The main thing to take note of compared to what I originally had, is that I completely disabled the drawer menu and removed the menu button entirely for V1. I didn't actually have a use for it at this point, however, in the future, there is definitely a spot to grow from.

(*I don't know Git well enough, but I know there are ways to squash commits, so I could do this beforehand.. I just personally don't care much*)

### CODE REVIEW

File: src/hooks/useMembers.testing.ts [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+      "avatarURL": "https://cdn.discordapp.com/avatars/217065422480998400/4ac7379ebf686b4c6b2d03547b5053d0.webp",
+      "discriminator": "1777",
+      "id": "217065422480998400",
+      "username": "Bomber"
+    }
+  }`);
+
+export default function useMembers(props?: Props): FG.State.MembersState {
+    const userIndex = useRef(0);
+    const { current: semaphore } = useRef<Semaphore>(new Semaphore(1));
```
Comment (dylhack) [inline src/hooks/useMembers.testing.ts:67]: A semaphore with a limit of 1, is this just a mutex?

Comment (demitchell14) [reply src/hooks/useMembers.testing.ts:67]: No, this is a good catch. 
It's the rate limiter for how many users can be fetched at one time.

Comment (demitchell14) [reply src/hooks/useMembers.testing.ts:67]: The actual limit that I have set on the `useMembers.ts` (not `useMembers.testing.ts`) is here..
https://github.com/Floor-Gang/modmail-web/blob/cleanup/public/i18n/en/translation.json#L5 - currently set to 2.

**Review by dylhack — APPROVED** [review]
Looks really good

**Pull request merged**